lib/model/autocomplete.dart

```diff
@@ -921,7 +921,7 @@ class TopicAutocompleteView extends AutocompleteView<TopicAutocompleteQuery, Top
   }
 
   TopicAutocompleteResult? _testTopic(TopicAutocompleteQuery query, TopicName topic) {
-    if (query.testTopic(topic)) {
+    if (query.testTopic(store, topic)) {
```

Comment:
autocomplete: Handle autocompleting to empty topic
    
    Signed-off-by: Zixuan James Li <zixuan@zulip.com>

This commit is NFC, right?